sys/conf/kern.mk

```diff
@@ -36,7 +36,7 @@ CWARNEXTRA?=	-Wno-error=tautological-compare -Wno-error=empty-body \
 		-Wno-error=pointer-sign
 CWARNEXTRA+=	-Wno-error=shift-negative-value
 CWARNEXTRA+=	-Wno-address-of-packed-member
-.if ${COMPILER_VERSION} >= 130000
+.if ${COMPILER_VERSION} >= 130000 && ${COMPILER_FEATURES:MWunused-but-set-variable}
```

Comment:
This is inconsistent, everywhere else replaces the version check. I'm in two minds which is best...

Reply:
Oh yeah, I initially added the check but then decided to replace and forgot to update this file. I don't mind either way but you are right I should be consistent.